### ISSUE TITLE
the label class numbers in tools are not accordance with the ground truth.

### ISSUE THREAD

**Issue by asimay** [wkentaro/labelme#160]
example: the color and label number is not same,
![label_viz](https://user-images.githubusercontent.com/8703449/39977267-4e867522-576b-11e8-9e84-97ef1a1bfd04.png)

but when generate ground truth, the class number is same.
![figure_1](https://user-images.githubusercontent.com/8703449/39977374-2a5ae20e-576c-11e8-9261-3779e3afbaaa.png)

**Comment by wkentaro** [comment]
This seems a similar problem as https://github.com/wkentaro/labelme/issues/52.
Could you provide below information and your json file for debug?

```
pip freeze
labelme --version
```

**Comment by asimay** [comment]
sorry for late.

> (lenet-env) E:\work\5_Project\deep-learning\labelme>pip freeze
absl-py==0.2.0
alabaster==0.7.10
asn1crypto==0.24.0
astor==0.6.2
astroid==1.6.0
Babel==2.5.3
backports.weakref==1.0rc1
bleach==1.5.0
bokeh==0.12.13
certifi==2017.11.5
cffi==1.11.4
chardet==3.0.4
click==6.7
cloudpickle==0.5.2
colorama==0.3.9
cryptography==2.1.4
cycler==0.10.0
dask==0.16.1
decorator==4.0.11
distributed==1.20.2
docutils==0.14
entrypoints==0.2.3
eventlet==0.22.0
Flask==0.12.2
Flask-SocketIO==2.9.6
future==0.16.0
gast==0.2.0
greenlet==0.4.12
grpcio==1.11.0
h5py==2.7.1
heapdict==1.0.0
html5lib==0.9999999
idna==2.6
imageio==2.1.2
imagesize==0.7.1
ipykernel==4.8.2
ipython==6.2.1
ipython-genutils==0.2.0
ipywidgets==7.1.2
isort==4.2.15
itsdangerous==0.24
jedi==0.11.1
Jinja2==2.10
jsonschema==2.6.0
jupyter-client==5.2.1
jupyter-console==5.2.0
jupyter-core==4.4.0
jupyterlab-launcher==0.10.2
Keras==1.2.1
labelme==2.13.0
lazy-object-proxy==1.3.1
locket==0.2.0
Markdown==2.6.9
MarkupSafe==1.0
matplotlib==2.1.1
mccabe==0.6.1
mistune==0.8.3
moviepy==0.2.3.2
msgpack-python==0.5.1
nbconvert==5.3.1
nbformat==4.4.0
networkx==2.0
notebook==5.3.1
numpy==1.14.0
numpydoc==0.7.0
olefile==0.44
pandas==0.22.0
pandocfilters==1.4.2
parso==0.1.1
partd==0.3.8
patsy==0.5.0
pickleshare==0.7.4
Pillow==5.0.0
prompt-toolkit==1.0.15
protobuf==3.5.2
psutil==5.4.3
pycodestyle==2.3.1
pycparser==2.18
pyflakes==1.6.0
Pygments==2.2.0
pylint==1.8.1
pyOpenSSL==17.5.0
pyparsing==2.2.0
PyQt5==5.10.1
PySocks==1.6.7
python-dateutil==2.6.1
python-engineio==2.0.1
python-socketio==1.8.3
pytz==2017.3
PyWavelets==0.5.2
pywinpty==0.5.1
PyYAML==3.12
pyzmq==16.0.3
QtAwesome==0.4.4
qtconsole==4.3.1
QtPy==1.3.1
requests==2.18.4
requests-toolbelt==0.8.0
rope==0.10.7
scikit-image==0.13.1
scikit-learn==0.19.1
scipy==1.0.0
seaborn==0.8.1
Send2Trash==1.4.2
simplegeneric==0.8.1
sip==4.19.8
six==1.11.0
snowballstemmer==1.2.1
sortedcontainers==1.5.9
Sphinx==1.6.6
sphinxcontrib-websupport==1.0.1
spyder==3.2.6
statsmodels==0.8.0
tblib==1.3.2
tensorboard==1.8.0
tensorflow==1.8.0
termcolor==1.1.0
terminado==0.8.1
testpath==0.3.1
Theano==1.0.1
toolz==0.9.0
tornado==4.5.3
tqdm==4.11.2
traitlets==4.3.2
typing==3.6.2
udacity-pa==0.2.8
urllib3==1.22
wcwidth==0.1.7
webencodings==0.5.1
Werkzeug==0.14.1
widgetsnbextension==3.1.4
win-inet-pton==1.0.1
win-unicode-console==0.5
wincertstore==0.2
wrapt==1.10.11
zict==0.1.3


> (lenet-env) E:\work\5_Project\deep-learning\labelme>labelme --version
> labelme 2.14.1

**Comment by wkentaro** [comment]
In your output of `pip freeze`, it says the version of labelme is old 2.13.0, so please upgrade it to the latest one.

```
Keras==1.2.1
labelme==2.13.0
lazy-object-proxy==1.3.1
```

**Comment by wkentaro** [comment]
Please request to reopen.
